lib/core/src/lib.rs

```diff
@@ -10,6 +10,9 @@
 // === Module Structure Reexport ===
 // =================================
 
+#[macro_use]
+extern crate shrinkwraprs;
```

Comment:
this is old rust style. use `use` instead. Or just import our `Prelude` instead :D